packages/console/src/components/MultiTextInput/utils.ts

```diff
@@ -0,0 +1,58 @@
+import { conditional } from '@silverhand/essentials';
+
+import { MultiTextInputError, MultiTextInputRule } from './types';
+
+const validate = (value: string[], rule?: MultiTextInputRule): MultiTextInputError | undefined => {
+  if (!rule) {
+    return;
+  }
+
+  const requiredError = conditional(
+    rule.required && value.filter(Boolean).length === 0 && rule.required
+  );
+
+  if (requiredError) {
+    return {
+      required: requiredError,
+    };
+  }
+
+  if (rule.pattern) {
+    const { regex, message } = rule.pattern;
+
+    const inputErrors = Object.fromEntries(
+      value.map((element, index) => {
+        return [index, regex.test(element) ? undefined : message];
+      })
+    );
+
+    if (Object.values(inputErrors).some(Boolean)) {
+      return {
+        inputs: inputErrors,
+      };
+    }
```

Comment:
since if the input value passes the regex validation it will always return an undefined. wondering if it is ok just always return the inputErrors.  No need to check if there is an actual error or not.

Reply:
I want to return an error or null. but the lint has a limit on the `null` usage. so use undefined instead.

Reply:
I mean always return an array.   No need to check some inputError exit. 

Always return 
```
     {
        inputs: inputErrors,
      };
```